page/sass/commit_funds.scss

```diff
@@ -47,13 +47,14 @@
       }
       .section-seperator{
         border-bottom: 1px $gray-extra-light solid;
-        width: 407px;
+        width: 90%;
```

Comment:
I think this is not the best way, to have this `.section-separator` fit to his parent, you can just solve the problem using html only, by moving this into the previous `div`, like this:
```
                <div class="content spacing">
                    <h3><%- sections[0].title -%></h3>
                        <p><%- sections[0].paragraph1 -%></p>
                        <p><%- sections[0].paragraph2 %></p>
                        <div class="section-seperator"></div>
                </div>
```
and then no need for `width: 90%`

🗡

Reply:
That's a good idea